Setup logic Selected

NG-GiaBao opened the pull request:
add BorderWhiteBox image
add BoardController script and funtions InitBoard

NG-GiaBao committed:
[add] BorderWhiteBox
[add] BoardController script and Funtions InitBoard

NG-GiaBao committed:
[Supplement] Main scene

ThucTranVN commented on Assets/MyGame/Scripts/Board/BoardController.cs:
```diff
@@ -0,0 +1,51 @@
+public class BoardController : BaseManager<BoardController>
+{
+    public GameObject tilePrefab;
+    public int width;
```
width and height should be private

ThucTranVN commented on Assets/MyGame/Scripts/Board/BoardController.cs:
```diff
@@ -0,0 +1,51 @@
+    {
+        string sizePart = boardType.ToString().Replace("Size", "");
+        string[] size = sizePart.Split('x');
+        width = int.Parse(size[0]);
```
width and height should be set by boardType
width = height = int(boardType)

NG-GiaBao committed:
[fix] add int enum boardType
[fix] add code check null in script BoardController
[fix] change tilePrefabs name

NG-GiaBao committed:
[Fix] Update code script BoardController và DataManager

NG-GiaBao committed:
[add] Object DataManager
[fix] Update script DataManager , GlobalConfig , BoardController

ThucTranVN reviewed (approved):
LGTM
